### ISSUE TITLE
Add Sockeye to VEBA deployment

### ISSUE THREAD

**Issue by embano1** [vmware-samples/vcenter-event-broker-appliance#320]
**Is your feature request related to a problem? Please describe.**
Add [Sockeye](https://github.com/n3wscott/sockeye) to VEBA to ease event inspection and troubleshooting.

**Describe the solution you'd like**
Access Sockeye UI after VEBA deployment, e.g. `https://${VEBA_IP}/events`

**Describe alternatives you've considered**
Tail event logs via CLI (as of today, e.g. `kn-echo`).

**Additional context**
n/a

**Comment by lamw** [comment]
Initial investigation integrating sockeye into VEBA lead to snag due to this this bug https://github.com/knative/eventing/issues/5139 which will be resolved in Eventing v0.22.0, which also does not currently have a release. We can consider updating Eventing once that's available

Some notes:

Create `sockeye.yaml`
```
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: sockeye
  name: sockeye
  namespace: vmware-system
spec:
  replicas: 1
  selector:
    matchLabels:
      app: sockeye
  template:
    metadata:
      labels:
        app: sockeye
    spec:
      containers:
      - image: docker.io/n3wscott/sockeye:v0.7.0@sha256:e603d8494eeacce966e57f8f508e4c4f6bebc71d095e3f5a0a1abaf42c5f0e48
        name: sockeye
        ports:
          - containerPort: 8080
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: sockeye
  name: sockeye
  namespace: vmware-system
spec:
  ports:
  - port: 80
    protocol: TCP
    targetPort: 8080
  selector:
    app: sockeye
  sessionAffinity: None
```

Update `ingressroute-gateway.yaml`
```
snip
...
  - conditions:
    - prefix: /events
    pathRewritePolicy:
      replacePrefix:
      - replacement: /
    services:
    - name: sockeye
      port: 80
  - conditions:
    - prefix: /static
    pathRewritePolicy:
      replacePrefix:
      - replacement: /static
    services:
    - name: sockeye
      port: 80
...
snip
```

**Comment by embano1** [comment]
The config for trigger/service looks good to me and is what I also validated with 0.22 in my local (kind) setup. The only difference is that I use Kourier as my Ingress, but should not be a deal breaker.
